src/content/changelog/4-11-1-2019-12-23.md

```diff
@@ -0,0 +1,11 @@
+---
+title: '4.11.1: 2020-01-13'
```

Comment:
Is this the correct date?

Reply:
yes--we previously were going to release a hotfix earlier, but then decided not to, and we kept this PR to store the changes for the Agnostic Front release this coming monday